org.eclipse.xtext.xbase.ui/src/org/eclipse/xtext/xbase/ui/contentassist/XbaseProposalProvider.java

```diff
@@ -743,6 +746,37 @@ public void completeXShortClosure_Expression(EObject model, Assignment assignmen
 	public void completeXMemberFeatureCall_Feature(EObject model, Assignment assignment, ContentAssistContext context,
 			ICompletionProposalAcceptor acceptor) {
 		if (model instanceof XMemberFeatureCall) {
+			XExpression memberCallTarget = ((XMemberFeatureCall) model).getMemberCallTarget();
+			IResolvedTypes resolvedTypes = typeResolver.resolveTypes(memberCallTarget);
+			LightweightTypeReference memberCallTargetType = resolvedTypes.getActualType(memberCallTarget);
+			Iterable<JvmFeature> featuresToImport = getFavoriteStaticFeatures(model, new Predicate<JvmFeature>() {
+				@Override
+				public boolean apply(JvmFeature input) {
+					if(input instanceof JvmOperation && input.isStatic()) {
+						List<JvmFormalParameter> parameters = ((JvmOperation) input).getParameters();
+						if(parameters.size() > 0) {
+							JvmFormalParameter firstParam = parameters.get(0);
+							JvmTypeReference parameterType = firstParam.getParameterType();
+							if(parameterType != null) {
+								return memberCallTargetType.isAssignableFrom(parameterType.getType());
```

Comment:
@holgerschill this is one of the problematic lines

Reply:
Here we should convert the JvmTypeReference to a lightweight type ref instead of unpacking the raw type.

Reply:
e.g. `memberCallTargetType.getOwner().toLightweightTypeReference(parameterType)`